cmd/oci-image-tool/validate.go

```diff
@@ -130,20 +143,30 @@ func (v *validateCmd) validatePath(name string) error {
 
 	f, err := os.Open(name)
 	if err != nil {
-		return errors.Wrap(err, "unable to open file")
+		return "", errors.Wrap(err, "unable to open file")
 	}
 	defer f.Close()
 
+	if v.verbose {
+		fd, err := os.Open(name)
```

Comment:
this doesn't seem the right place to re-open the file just to get its digest, I would have done this inside each validation func (and as already explained, on every object being validated)